canonical-web-and-design/maas-ui
# fix(ui): fix settings spinner alignment

## Done

- Fix the alignment of the spinner in the settings tables.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to settings and click on a table that has a lot of items (scripts, DHCP snippets etc.)
- You should see the loading spinner in the centre of the table.

## Fixes

Fixes: #2427.
